ui/ui_test.go

```diff
@@ -0,0 +1,33 @@
+package pitchforkui
+
+func ExamplePfUI_Page_show() {
```

Comment:
Commit: https://github.com/tridentli/pitchfork/pull/111/commits/cf63c3f7c2d3352967f75296bad708472a03a75e

MR that has this: https://github.com/tridentli/pitchfork/pull/111

Note that this also needs: https://github.com/tridentli/trident/pull/84